plugins/generator-1.18.2/forge-1.18.2/features/feature_coral_tree.java.ftl

```diff
@@ -0,0 +1 @@
+NoneFeatureConfiguration.INSTANCE
```

Comment:
Instead of having multiple files with the same code, maybe you should simply combine them in a single file/feature for all corals.

Reply:
Not possible, as the features use different classes